app/helpers/courses_helper.rb

```diff
@@ -19,7 +19,8 @@ def show_new_course_button_if_admin
   end
 
   def degree_requirement_for(course)
-    course.degree_requirement.humanize.downcase
+    requirement_name = course.degree_requirement.humanize.downcase
+    requirement_name.sub(/ /, '-')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.